app/components/user-avatar.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import config from 'travis/config/environment';
+import {gravatarImage} from '../utils/urls';
+
+export default Ember.Component.extend({
+
+  tagName: 'span',
+  classNames: ['avatar'],
+
+  gravatarUrl: function() {
+    if (!this.get('isAuthor') && !this.get('isComitter')) {
+      if (this.get('user.type') === 'organization') {
+        return this.get('user.avatarUrl');
+      } else {
+          return gravatarImage(this.get('user.email'), 36);
```

Comment:
Can't we use `user.avatarUrl` here as well? I thought that we now save `avatarUrl` from GitHub for all type of accounts.